nvme.spec.in

```diff
@@ -5,7 +5,6 @@ Summary:  	Core nvme tools
 License: 	GPL
 Group: 		Development/Tools
 URL: 		https://github.com/linux-nvme/nvme-cli/
-Source: 	nvme-@VERSION@.tar.gz
 Provides:	nvme
 Requires(post): util-linux systemd systemd-udev
 BuildRoot:	%{_tmppath}/%{name}-%{version}-root
```

Comment:
I think it's good idea to write that build requires explicitly.

so do you want that to build with lib that already has own rpm version
by explicitly write the lib pkgs?

Reply:
Actually, I don't use the spec file generated by libnvme for openSUSE. I don't know how other distro handle this. Fedora also ignores this generated spec file. Given we just removed the support to generated a deb package by dropping the Makefile, we might just remove this spec file as well. What's the point if no one is using it?

Reply:
None of the distros use it, AFAIK. My original intention was for a developer to make a bench build package to distribute for internal testing. I used to use it frequently, but not anymore. If no one's using it, then that's fine, we can remove it.

Reply:
I assumed this was the case. I mean I don't have problem with keeping it but in this case we know the target audience.

BTW, I was able to build a almost completely statically linked version of nvme-cli. The only thing which was not linked in was the libc. So basically you can just hand over such a binary to a tester if the glibc match :)

Reply:
Meson wrapped subprojects help out there, I assume?

Reply:
Yes, I think this is what is in the spirit of the initial spec file. It should be something for quick test deployment. So it should be as self contained as possible:

`meson setup --wrap-mode=forcefallback --default-library=static .build `

That would give a binary with only one additional dependency (libuuid) except libc. I haven't found a way to build a complete static library. Don't know if this is supported.

Reply:
You would presumably just add `-static` to your LDFLAGS?

Reply:
Ah that's the trick. Seems to do work, as expected there is some fallout around libuuid. I suppose I should try to get libuuid into wrapdb :)

Reply:
util-linux has an upstream meson.build, but it doesn't seem to provide a `declare_dependency()` to reference in a WrapDB `util-linux.wrap` inside a `[provide]` section. So that would be a good place to start. :)

Reply:
Well, it's not a quick win. Got some progress but there are a bunch of issues to solve until this will fly. I'll cleanup my first attempt and send out a PR for discussion.

Reply:
As @keithbusch mentioned, I am using it for internal test.
But I don't mind removing this rpm spec.
If you guys wants to remove, then I will close this PR and make new PR that remove spec(rpm) & control(deb).